apps/app/src/mocks/miragejs/orm/db/DbCollection.ts

```
@@ -0,0 +1,329 @@
+      ([key, value]) => String(record[key as keyof DbRecord<TAttrs, TId>]) === String(value),
+    );
+  }
+
+  /**
+   * Prepares a record for insertion by generating an ID if it doesn't exist.
+   * @param data - The record to prepare.
+   * @returns The prepared record.
+   */
+  private _prepareRecord(data: DbRecordInput<TAttrs, TId>): DbRecord<TAttrs, TId> {
+    const record = { ...data } as DbRecord<TAttrs, TId>;
+    if (!record.id) {
```

ESLint

[error] Unexpected negated condition.

(unicorn/no-negated-condition)